docs/api/source/index.rst

```diff
@@ -16,17 +16,21 @@ a container.
 .. admonition:: Supported Protocols
    :class: caution
 
-   This library currently only supports an `early version of WebSocket
+   This library currently only supports an Inrupt `Enterprise Solid Server
```

Comment:
would remove "only".

Reply:
Well, there are in theory other specs & types (e.g., webhooks or polling)

Reply:
Actually, looking at the sentence, maybe even remove "currently only".  The reason, not because there are or there aren't other things, but

- People don't want to read reference docs.  Any extra word or two just prolongs a task that people would prefer not to do.
- They just want to know if a particular thing does what they need to do. 

As such, with the following sentences:

- This library supports blah
- This library currently supports blah. 
- This library currently only support blah.

The key point for end-users of this library is "This library supports blah."

- If my project doesn't use blah, job done.  The sentence tells me I need to look elsewhere.
- If my project uses blah, also, job done.  The sentence tells me that I can use this library. 

On the surface, sentences like " This library currently only support blah." seems like it would help people with a little more info, but in effect, it doesn't.  End-users can't tell when the library will support something else or what the something else is.  The sentence basically states "This library supports blah.  This library in the future someday could support something else, but whether it will or when or what, ..."  As such, usually in reference writing, we do our best to get people in and out.

Reply:
I think the assumption is that a user may know the spec and be like "what about webhooks? Or solid-0.1?"

Reply:
Tee hee. I wasn't kidding about my mind being aflutter.  I meant the below paragraph.  And yay, the person is here building my desk.

Once we say we support ESS version of WebSocket notifications, do we need to say more? From that, I think people will know that we don't support webhooks or solid-0.1?  But, despite all the writing I did, I actually don't have strong opinions. It's just that from past, I found it much easier to maintain smaller number of sentences than larger numbers 😄

Reply:
And lol ... good grief, I am getting my comments all mixed up.  This is why there's someone putting together my Ikea desk rather than my taking out the electric drill.  Am going to have to give myself a little timeout in the corner.